extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -139,11 +141,18 @@ describes.repeated('', {
           form.appendChild(emailInput);
 
           ampForm.method_ = 'GET';
-          ampForm.xhrAction_ = null;
           sandbox.stub(form, 'submit');
           sandbox.stub(form, 'checkValidity').returns(true);
           sandbox.stub(ampForm, 'analyticsEvent_');
           sandbox.stub(ampForm.ssrTemplateHelper_, 'isSupported').returns(true);
+
+          return Promise.resolve(ampForm);
```

Comment:
nit: You can return the value `ampForm` itself. `Promise.resolve` is unnecessary inside a Promise chain.

Reply:
done.